README.md

```diff
@@ -1,92 +1,199 @@
-|Intel(R) DAAL|2020 Gold|[rls/daal-2020-rls](https://github.com/oneapi-src/oneDAL/tree/rls/daal-2020-rls)</br>[rls/daal-2020-mnt](https://github.com/oneapi-src/oneDAL/tree/rls/daal-2020-mnt) (contains ongoing fixes)|&nbsp;&nbsp;&nbsp;[Home page](https://software.intel.com/en-us/daal)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[Developer Guide](https://software.intel.com/en-us/daal-programming-guide)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[System Requirements](https://software.intel.com/en-us/articles/intel-data-analytics-acceleration-library-2020-system-requirements)|
+Intel&reg; DAAL K-means fit, strong scaling result | Intel&reg; DAAL K-means fit, weak scaling results
+:-------------------------:|:-------------------------:
+![](https://github.com/PivovarA/daal/raw/dev/apivovar-newReadme/.github/charts/Intel%20DAAL%20KMeans%20strong%20scaling.png)  |  ![](https://github.com/PivovarA/daal/raw/dev/apivovar-newReadme/.github/charts/intel%20DAAL%20KMeans%20weak%20scaling.png)
 
-## Installation
+## Python API
 
-You can install oneDAL: 
+[daal4py](https://github.com/IntelPython/daal4py) is an Intel&reg; DAAL python API.
+Below is an example of how daal4py can be used for calculation KMeans clusters
 
-- from [oneDAL home page](https://software.intel.com/en-us/oneapi/onedal) as a part of Intel(R) oneAPI Base Toolkit.
-- from [GitHub\*](https://github.com/oneapi-src/oneDAL/releases).
+```python
+import numpy as np
+import pandas as pd
+import daal4py as d4p
 
-See [Installation from Sources](INSTALL.md) for details.
+data = pd.read_csv("local_kmeans_data.csv", dtype = np.float32)
+
+init_alg = d4p.kmeans_init(nClusters = 10,
+                           fptype = "float",
+                           method = "randomDense")
+
+centroids = init_alg.compute(data).centroids
+alg = d4p.kmeans(nClusters = 10, maxIterations = 50, fptype = "float",
+                 accuracyThreshold = 0, assignFlag = False)
+result = alg.compute(data, centroids)
+```
+
+Often data scientists require different tools for analysis regular and big data. daal4py offers various processing models, which makes it easy to enable distributed multi-node mode.
+```python
+import numpy as np
+import pandas as pd
+import daal4py as d4p
+
+d4p.daalinit() # <-- Initialize SPMD mode
+data = pd.read_csv("local_kmeans_data.csv", dtype = np.float32)
+
+init_alg = d4p.kmeans_init(nClusters = 10,
+                           fptype = "float",
+                           method = "randomDense",
+                           distributed = True) # <-- change model to distributed
+
+centroids = init_alg.compute(data).centroids
+
+alg = d4p.kmeans(nClusters = 10, maxIterations = 50, fptype = "float",
+                 accuracyThreshold = 0, assignFlag = False,
+                 distributed = True)  # <-- change model to distributed
+
+result = alg.compute(data, centroids)
+```
+
+## Scikit-learn patching
```

Comment:
The context is not aligned with the title

Reply:
Not addressed yet.

Reply:
I moved scikit-learn patching paragraph to Python API paragraph and added explanation: `Python interface to efficient Intel® oneDAL provided by daal4py allows one to create scikit-learn compatible estimators, transformers, clusterers, etc. powered by oneDAL which are nearly as efficient as native programs.`